book/src/nostr-sdk/03-quickstart.md

```diff
@@ -0,0 +1,34 @@
+## Quickstart
+
+<custom-tabs category="lang">
+
+<div slot="title">Rust</div>
+<section>
```

Comment:
Can you add the tabs for all the supported languages? You can find the template in `book/contrib/content-tabs-template.md`. It's OK to leave the `TODO` if you not add the examples for all languages.